README.md

```diff
@@ -1,5 +1,28 @@
 # T-asapp
 Proyecto ingeniería software
 
-PROYECTO DE GESTIÓN LOGÍSTICA DE ENVÍO DE PAQUETERÍA
+PROYECTO DE GESTIÓN LOGÍSTICA DE ENVÍO DE PAQUETERÍA A CENTROS EDUCATIVOS DE ANDALUCÍA
+
+Este proyecto pertenece a la materia de Ingeniería del Software del Master en Programación y Diseño de Aplicaciones
+de EDIAE y T-Systems.
+
+El proyecto parte de la necesidad de gestionar un sistema de paquetería orientado a facilita a empresas de paquetería
+el envío de diferntes tipos de productos y paquetesa centros educativos de Andalucía. 
+
+El cliente necesita poder gestionar dichos envíos envíos, bien de documentación o material teniendo 
```

Comment:
Aparte del error de repetición de una palabra, no estáis contando el problema, sino la solución: gestionar envíos. Si colocáis el problema antes que la solución, cualquier programa que haga eso estará "bien", sin que sin embargo resuelva en realidad ningún problema, simplemente porque no sabemos cuál es ese problema.